### PR TITLE
Feature/2.0.5

### PR DESCRIPTION
**Pull request by malkafly-cw**
- New guided Pix UI on checkout
- Fixing the discount value on PIX

### CODE REVIEW

File: templates/welcome/welcome.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
                     <iframe width="560" height="315" src="https://www.youtube.com/embed/-T85G-RYTFE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                 </p>
 
-                <p><a class="button" target="_blank" href="https://metabox.io/online-generator/?utm_source=WordPress&amp;utm_medium=link&amp;utm_campaign=plugin">Para gerar suas credências clique aqui</a></p>
+                <p><a class="button" target="_blank" href="#">Para gerar suas credências clique aqui</a></p>
```
Comment (stephania-cw) [inline templates/welcome/welcome.php:21]: credenciais, sem acento

File: templates/checkout/checkout.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 	$i <= (int) $max_installments;
 	$i ++
 ) {
-	if ( $i === 1 ) {
+	if ( $i == 1 ) {
 		$installments .= '<option value="1">R$ ' . esc_attr( number_format( $installments_value[ $i - 1 ]['value'], 2, ",", "." ) ) . ' à vista</option>';
 	} else {
 		$new_value = round( $installments_value[ $i - 1 ]['value'], 2, PHP_ROUND_HALF_UP );
```
[the remark this answers is not in the document]
Comment (malkafly-cw) [reply templates/checkout/checkout.php:24]: it's possible! I will fix the code

**Review by panachi-cw — APPROVED** [review]
🙏

**Review by bru-carenzi — APPROVED** [review]
### Test Summary

**Status**: 👍 Ready


---


📋 Tests:

*  Pix payment - validated on prod
*  Pix payment with discount  - validated
*  Pix new interface - validated
*  Credit card payment - validated on prod
*  Credit card payment in installments - validated on stg
*  Check the order status after payment - validated
*  Cancel a purchase on App - validated with cw merchant
*  All the bug fix were validated

**Review by panachi-cw — APPROVED** [review]

**Pull request merged**